docs/automation-example.md

```diff
@@ -0,0 +1,69 @@
+# example workflow
+## Detecting new ECR images and/or Lambda functions in AWS and importing/testing them with Snyk
+Using AWS's services, specifically EventBridge, you can now detect almost any processes from other services, including ECR and Lambda, and trigger a Lambda function to be invoked upon that detection.
```

Comment:
link to `EventBridge` page would be nice

Reply:
Done